README.md

```diff
@@ -85,6 +90,7 @@ None
   roles:
     - ansible-role-fluentd
   vars:
+    fluentd_flags: "{% if ansible_os_family == 'FreeBSD' %}-p {{ fluentd_plugin_dir }}{% elif ansible_os_family == 'Debian' %}{% elif ansible_os_family == 'RedHat' %}{% endif %}"
```

Comment:
there are two way to manage plug-ins: one is using `fluent-gem` and another is placing plug-in files under `plugin` directory.  `fluent-gem` does not install plug-ins to `plugin` directory. `plugin` directory in question is for those plug-ins that are locally-brewed. the former requires you to create a gem, on the other hand, the latter does not. See:
http://docs.fluentd.org/v0.12/articles/plugin-management#add-a-plugin-via-etcfluentplugin